sympy/conftest.py

```diff
@@ -7,8 +7,11 @@
 from sympy.external.gmpy import GROUND_TYPES, HAS_GMPY
 from sympy.utilities.misc import ARCH
 import re
+from hypothesis import settings
 
 sp = re.compile(r'([0-9]+)/([1-9][0-9]*)')
+settings.register_profile("no_deadline", deadline=None)
```

Comment:
We might as well call this `sympy_hypothesis_profile`. I expect we'll want to modify other settings as well going forward.